### PR TITLE
Change name of "Parameterised Measurement Result"

### LINKED ISSUE #450: Different Naming of "ParameterizisedMeasurementResult"
In the code it is called `ParameterisationDepentMeasurementResult` and in the Diagram `ParameterizisedMeasurementResult`.

Which one do you prefer?

### PR DESCRIPTION
**Pull request by acBerger**
fixes #450

[Review on Reviewable](https://reviewable.io/reviews/beagle-pse/beagle/452)

### CODE REVIEW

**Comment by RomanLangrehr** [comment]
:lgtm_strong: 

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/beagle-pse/beagle/452#-:-K7MdOwk8fHX8lyuZZ-s:-1868698381)*

File: doc/Design/Packages/Measurement.uml [hunk 1]
```diff
@@ -33,7 +33,7 @@
       </ownedAttribute>
     </packagedElement>
     <packagedElement xmi:type="uml:Class" xmi:id="_Y4Hu8LJHEeWgdqdZ8hPRgg" name="Parametrisation"/>
-    <packagedElement xmi:type="uml:Class" xmi:id="_Yae8ALLcEeWYbvIJ2tLPPw" name="Parameterised Measurement Result" isAbstract="true">
+    <packagedElement xmi:type="uml:Class" xmi:id="_Yae8ALLcEeWYbvIJ2tLPPw" name="Parameterisation Dependent Measurement Result" isAbstract="true">
```
Comment (m1cm1c) [inline doc/Design/Packages/Measurement.uml]: "Parameterisation-Dependent" is missing a hyphen.

**Comment by acBerger** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[doc/Design/Packages/Measurement.uml, line 36 \[r1\]](https://reviewable.io:443/reviews/beagle-pse/beagle/452#-K7MdB8rirNZxOtZTRht-r1-36:-K7MeMhUGql41SkcAkG5:-499146009) ([raw file](https://github.com/beagle-pse/beagle/blob/405b3edebdf780dd1a1b79281feee11d25a10f1a/doc/Design/Packages/Measurement.uml#L36)):*
Done.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/beagle-pse/beagle/452)*

**Comment by m1cm1c** [comment]
:lgtm_strong: 

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/beagle-pse/beagle/452#-:-K7MfJwN1qXCsAyOx9tM:-1868698381)*

**Comment by RomanLangrehr** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks pending.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/beagle-pse/beagle/452)*

**Pull request merged**